src/remote/impl/rms.external.controller.service.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpService } from '@nestjs/common';
+import { Logger } from 'protocol-common/logger';
+import { ProtocolHttpService } from 'protocol-common/protocol.http.service';
+import { IExternalControllerService } from '../external.controller.service.interface';
+
+export class RmsExternalControllerService implements IExternalControllerService {
+
+    private readonly http: ProtocolHttpService;
+
+    /**
+     * We pass in the parent class as context so we can access the sms module
+     */
+    constructor(
+        httpService: HttpService,
+    ) {
+        this.http = new ProtocolHttpService(httpService);
+    }
+
+    public async callExternalWalletCreate(identityNumber: string): Promise<string> {
+        // for honduras this is NumeroIdentidad
```

Comment:
This comment doesn't belong here (implementation-specific things don't belong in this repo).

Reply:
done